pkg/kubectl/describe.go

```diff
@@ -545,7 +545,7 @@ func describePod(pod *api.Pod, events *api.EventList) (string, error) {
 			}
 		}
 		describeVolumes(pod.Spec.Volumes, out, "")
-		fmt.Fprintf(out, "QoS Class:\t%s\n", qos.GetPodQOS(pod))
+		fmt.Fprintf(out, "QoS Tier:\t%s\n", qos.GetPodQOS(pod))
```

Comment:
> its breaking the upgrade test for kubectl skew which runs the 1.3 unit tests on a 1.4 cluster, basically the command line output changed between versions and now the upgrade tests are broken, I'm unsure what the usual protocol is here.

I'd prefer to make the output matching test for the describer more tolerant.  @ixdy can we do that instead?  The describer isn't an API and we should have the freedom to change its output between versions to be more useful/usable.